npm-frontend/src/ErrorPage.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import Logo from './Logo'
+
+const ReturnHomeButton = styled.button`
+  font-weight: bold;
+  color: #2A77C9;
+  height: 90px;
+  width: 190px;
+  border-radius: 8px;
+  font-size: 25px;
+  cursor: pointer;
+  margin: 15px;
+  background-color: #EBEBEB;
+  border-color: #272727;
+  box-shadow: inset 0px 0px 21px 0px #272727, 13px 12px 20px -5px black;
+  transition: width .5s, height .5s, font-size .5s, background-color .5s;
+
+  &:hover {
+    height: 105px;
+    width: 220px;
+    font-size: 29px;
+    background-color: #EBEBF1;
+  }
+`;
+
+const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 150px;
```

Comment:
Reduce the padding on the top here because right now I can't see the entire page at once when it loads. The button is mostly cutoff on my screen. 50px might be a better amount to try, but just play around with it and see what you think!